internal/api/api.proto

```diff
@@ -29,8 +29,21 @@ message StateRequest {}
 message StateResponse {State state = 1;}
 
 /* Config */
-message WriteSettingsRequest {}
-message WriteSettingsResponse {}
+message MinecraftConfig {
+    optional bool hardcore = 1;
+    optional int32 max_players = 2 [json_name = "max-players"];
+    optional string motd = 3;
+}
+
+message UpdateConfigRequest {
```

Comment:
if i understand that correctly to "create" config (actually init), we will use update too